OD-594 Add rating to stories

hellingp opened the pull request:
This adds a rating column/field to the "stories" table because step 2 of ODAP needs it.

hellingp committed:
Add rating to stories

This adds a rating column/field to the "stories" table because step 2 of ODAP needs it.

ariana-paris commented on opendoors/open-doors-tables-working.sql:
```diff
@@ -114,6 +114,7 @@ CREATE TABLE IF NOT EXISTS `stories`
     `do_not_import` tinyint(1)   NOT NULL DEFAULT '0',
     `ao3_url`       varchar(255)          DEFAULT NULL,
     `import_notes`  varchar(1024)         DEFAULT '',
+    `rating` 		tinyint(4)	 NOT NULL DEFAULT 0,
```
I don't think we actually store anything in this field in the eFiction code (or we'd have noticed it was missing sooner!), so this might as well start off life as it will eventually end it, as a varchar big enough to hold `Teen And Up Audiences` (ie [the Archive's 5 official ratings](https://archiveofourown.org/help/symbols-key.html))

hellingp replied:
Yeah, I see how I didn't think this through all the way. Should there be a default rating?

ariana-paris replied:
I think it's fine to leave it empty as having a NULL in the field will make it clearer if there's a processing error further down the process when we want to put the original ratings in.

ariana-paris reviewed (commented):
Looks good, though I think we can go for a better type and you need to update the test that's failing so the data it's expecting includes the new field (see the diff in the Pytest section in the Linux Github Action)

hellingp committed:
Add rating to stories

Adds a rating column to stories. Rating defaults to NULL. Also updates tests. The original rating was a foreign key; this commit does not supply each story with a new rating.

ariana-paris committed:
OD-594 Fix missing rating field in tests
